client/verta/verta/deployment/autoscaling/metrics.py

```diff
@@ -7,5 +7,36 @@
 
 @six.add_metaclass(abc.ABCMeta)
 class _AutoscalingMetric(object):
-    def _as_json(self):
-        raise NotImplementedError
+    _METRIC_ID = 0
+    _NAME = ""
+
+    def __init__(self, value):
+        self._value = value
```

Comment:
```suggestion
        self._value = str(value)
```

Reply:
Oh yeah the conversion was in the deleted code.

Reply:
Fixed.